app/services/users/profile.rb

```diff
@@ -58,7 +58,7 @@ def fetch_and_serialize_profile
     end
 
     def account
-      { account_uuid: user.account_uuid }
+      { account_uuid: user.account_uuid, created_at: user.initial_sign_in }
```

Comment:
Could you actually move this attribute into the `profile` section? Just a line like:
`initial_sign_in: user.initial_sign_in`